### PR TITLE
Training/03 magento UI

### PR DESCRIPTION
**Pull request by jlboban**

### CODE REVIEW

File: app/code/Inchoo/Sample03/view/frontend/layout/inchoo_sample03_index_weight.xml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0"?>
+
+<page xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd">
+    <body>
+        <referenceContainer name="content">
+            <block name="inchoo_sample03_weight" class="Inchoo\Sample03\Block\Weight" template="Inchoo_Sample03::custom/weight.phtml">
```
Comment (tnikcevs) [inline app/code/Inchoo/Sample03/view/frontend/layout/inchoo_sample03_index_weight.xml:6]: nepotrebno imati definiran i kreiran block ako nema posebne logike u njemu, viewmodel dovoljan

**Review by tnikcevs — CHANGES_REQUESTED** [review]

File: app/code/Inchoo/Sample03/view/frontend/templates/custom/weight.phtml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+/** @var \Magento\Framework\View\Element\Template $block */
+/** @var \Magento\Framework\Escaper $escaper */
+/** @var Inchoo\Sample03\ViewModel\Weight $weightViewModel */
+$weightViewModel = $block->getData('view_model');
+?>
+
+<div class="weight-list-container">
+    <ul class="weight-list">
+        <?php foreach ($weightViewModel->getWeightUnits() as $weight) : ?>
+            <?php // phpcs:ignore ?>
```
Comment (ivanmatozan) [inline app/code/Inchoo/Sample03/view/frontend/templates/custom/weight.phtml:11]: za čega je ovaj ignore?

File: app/design/frontend/Inchoo/custom/Inchoo_Sample03/layout/inchoo_sample03_index_index.xml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<layout xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_layout.xsd">
+    <update handle="inchoo_sample03_index_index"/>
```
Comment (ivanmatozan) [inline app/design/frontend/Inchoo/custom/Inchoo_Sample03/layout/inchoo_sample03_index_index.xml:2]: ovaj update ti ne treba, update ti je kao "include" za layoute

**Review by ivanmatozan — CHANGES_REQUESTED** [review]

File: app/code/Inchoo/Sample03/view/frontend/templates/custom/weight.phtml (hunk 2 above)
Comment (jlboban) [reply app/code/Inchoo/Sample03/view/frontend/templates/custom/weight.phtml:11]: jure@jure-G752VM:~/docker/m2$ codecheck

FILE: ...code/Inchoo/Sample03/view/frontend/templates/custom/weight.phtml
----------------------------------------------------------------------
FOUND 0 ERRORS AND 1 WARNING AFFECTING 1 LINE
----------------------------------------------------------------------
 11 | WARNING | Variable $escaper is undefined.
    |         | (VariableAnalysis.CodeAnalysis.VariableAnalysis.UndefinedVariable)
----------------------------------------------------------------------

**Review by tnikcevs — APPROVED** [review]

**Review by ivanmatozan — APPROVED** [review]

**Pull request merged**